### DISCUSSION TITLE
What the problem in 'Layering' blending, use one layerd blend per bone to merge parts?

### DISCUSSION THREAD

**Discussion by watsonsong** [Sixze/ALS-Refactored#536] [Q&A]
![image](https://github.com/user-attachments/assets/0c011b7a-b153-442b-a8a1-1de0be2deeed)
There has some layerd blend per bone node not in use. And the comment says they was not working correctly because the bug of unreal engine. What's the problem about it? And if I switch this way, will it be better performance?

**Comment by Sixze** [comment]
I've updated this comment with more details. Basically, the rifle does not stay in line with the camera when aiming while moving, and something weird happens to fingers when rolling with the rifle.
